frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/SelectedTaskDrawerContent.tsx

```diff
@@ -36,10 +29,10 @@ const SelectedTaskDrawerContent: React.FC<SelectedTaskDrawerContentProps> = ({ t
           <DrawerCloseButton onClick={onClose} />
         </DrawerActions>
       </DrawerHead>
-      <DrawerPanelBody>
+      <DrawerPanelBody className="pf-v6-u-p-md">
```

Comment:
CSS Classes should be opaque to Dashboard -- I do not like this, but understand its a solution for the use-case. I hope PF is aware of this need/gap in their `DrawerPanelBody` @jenny-s51

Reply:
Thank you for your review and feedback @andrewballantyne - it does indeed resolve the current use case though it is not ideal. 

I've created a [PF ticket to resolve this gap](https://github.com/patternfly/patternfly-react/issues/11804) and linked it in both files that apply this utility class.